hikari/api/rest.py

```diff
@@ -727,31 +727,31 @@ async def create_message(
             content is instead treated as an attachment if no `attachment` and
             no `attachments` kwargs are provided.
         embed : hikari.undefined.UndefinedOr[hikari.embeds.Embed]
-            If specified, the message embed.
+            If provided, the message embed.
         attachment : hikari.undefined.UndefinedOr[hikari.files.Resourceish],
-            If specified, the message attachment. This can be a resource,
+            If provided, the message attachment. This can be a resource,
             or string of a path on your computer or a URL.
         attachments : hikari.undefined.UndefinedOr[typing.Sequence[hikari.files.Resourceish]],
-            If specified, the message attachments. These can be resources, or
+            If provided, the message attachments. These can be resources, or
             strings consisting of paths on your computer or URLs.
         tts : hikari.undefined.UndefinedOr[builtins.bool]
-            If specified, whether the message will be TTS (Text To Speech).
+            If provided, whether the message will be TTS (Text To Speech).
         nonce : hikari.undefined.UndefinedOr[builtins.str]
-            If specified, a nonce that can be used for optimistic message
+            If provided, a nonce that can be used for optimistic message
             sending.
         mentions_everyone : hikari.undefined.UndefinedOr[builtins.bool]
-            If specified, whether the message should parse @everyone/@here
+            If provided, whether the message should parse @everyone/@here
             mentions.
         user_mentions : hikari.undefined.UndefinedOr[typing.Union[typing.Collection[hikari.snowflakes.SnowflakeishOr[hikari.users.PartialUser]], builtins.bool]]
-            If specified, and `builtins.True`, all mentions will be parsed.
-            If specified, and `builtins.False`, no mentions will be parsed.
+            If provided, and `builtins.True`, all mentions will be parsed.
```

Comment:
I don't like this wording. Parsed to...what?

We should say something like `If provided and builtins.True, all types of mentions will be detected. If builtins.False, all mention types will ignored if appearing in the message body.` or something a bit clearer, imho.